packages/clay-select/src/ClaySelect.soy

```diff
@@ -0,0 +1,73 @@
+{namespace ClaySelect}
+
+/**
+ * This renders the component's whole content.
+ */
+{template .render}
+	{@param options: list<?>}
+	{@param? disabled: bool}
+	{@param? id: string}
+	{@param? inline: bool}
+	{@param? label: string}
+	{@param? multiple: bool}
+	{@param? name: string}
+
+	<div class="form-group" id="{$id}">
+		{if $label}
+			<label>{$label}</label>
+		{/if}
+
+		{let $select kind="html"}
+			{call .select}
+				{param disabled: $disabled /}
+				{param multiple: $multiple /}
+				{param name: $name /}
+				{param options: $options /}
+			{/call}
+		{/let}
+
+		{if $inline}
+			<div class="form-inline">
```

Comment:
Hey @carloslancha 

I believe the `form-inline` class is intended for a wrapper element like the `<form>` tag. I think we could possibly use the `form-inline-group` class on the root element instead.

@pat270 is working on updating all the markup documentation on the [claycss.com](https://github.com/liferay/claycss.com) repo, once he does we'll make it available internally so stuff like this is easier to find.